03_Modules/EVDriver/src/module/api.ts

```diff
@@ -137,6 +139,19 @@ export class EVDriverModuleApi
       );
   }
 
+  @AsDataEndpoint(
+    Namespace.LocalListVersion,
+    HttpMethod.Get,
+    ChargingStationKeyQuerySchema,
+  )
+  async getLocalAuthorizationList(
```

Comment:
How about renaming this to `getLocalAuthorizationListVersion`?

Reply:
That makes more sense. I couldn't name it 'getLocalListVersion' since that's already a message api endpoint, but this works. Fixed!